bears/vcs/git/GitCommitBear.py

```diff
@@ -213,23 +213,34 @@ def check_imperative(self, paragraph):
     def check_body(self, body,
                    body_line_length: int=72,
                    force_body: bool=False,
-                   ignore_length_regex: typed_list(str)=()):
+                   ignore_length_regex: typed_list(str)=(),
+                   body_regex: str=None):
         """
         Checks the given commit body.
 
-        :param body:                The commit body splitted by lines.
+        :param body:                The body of the commit message of HEAD.
         :param body_line_length:    The maximum line-length of the body. The
                                     newline character at each line end does not
                                     count to the length.
         :param force_body:          Whether a body shall exist or not.
         :param ignore_length_regex: Lines matching each of the regular
                                     expressions in this list will be ignored.
+        :param body_regex:          If provided, checks the presence of regex
+                                    in the commit body.
         """
         if len(body) == 0:
             if force_body:
                 yield Result(self, 'No commit message body at HEAD.')
             return
 
+        if body_regex:
+            if not re.search(body_regex, body):
+                yield Result(self, 'No match found for the regular '
+                                   'expression provided: %s' % body_regex)
```

Comment:
-> No match found **in commit message** for the regular expression provided:

People could misunderstand where this message comes from^^